src/p2p/net_node.inl

```diff
@@ -1569,7 +1574,7 @@ namespace nodetool
             }
             // continue for another few cycles
           }
-          else
+          else if (!is_connected_to_at_least_one_seed_node)
```

Comment:
Do you want to suppress the warning AND `break`? Seems like only the log statement should be suppressed here?

Reply:
You're quite right, fixing.